package.json

```diff
@@ -3,8 +3,10 @@
   "version": "3.2.1",
   "description": "Some tools for docker",
   "main": "lib/docker-toolbelt.js",
+  "types": "lib/docker-toolbelt.d.ts",
   "scripts": {
-    "prepublish": "npm run test && coffee -c lib",
+    "prepublishOnly": "npm run test && coffee -c lib",
```

Comment:
I think it makes sense to remove the ` && coffee -c lib` here and add `npm run build` to the `test` script